src/AutoRecordings.cpp

```diff
@@ -172,22 +172,22 @@ PVR_ERROR AutoRecordings::SendAutorecAdd(const PVR_TIMER &timer)
   if (strcmp(timer.strDirectory, "/") != 0)
     htsmsg_add_str(m, "directory", timer.strDirectory);
 
-  /* Note: not sending any of the following for values will be interpreted by tvh as "any". */
+  /* Note: not sending any of the following three values will be interpreted by tvh as "any". */
   if (timer.iClientChannelUid >= 0)
     htsmsg_add_u32(m, "channelId", timer.iClientChannelUid);
 
   if (timer.startTime > 0)
   {
-    /* approx start time in minutes from midnight. tvh will match value +- 15 mins. */
+    /* Exact start time (minutes from midnight). */
     struct tm *tm_start = localtime(&timer.startTime);
-    htsmsg_add_s32(m, "approxTime", tm_start->tm_hour * 60 + tm_start->tm_min);
+    htsmsg_add_s32(m, "start", tm_start->tm_hour * 60 + tm_start->tm_min);
```

Comment:
When using approxtime, tvh will records shows starting 15 mins before or after this.

If we set "start" to our actual start time, there will be no margin, if a broadcaster decides to show it 5 min in advance, the show will not be recorded :s